core/src/test/scala/unit/kafka/zk/EmbeddedZookeeper.scala

```diff
@@ -54,6 +53,12 @@ class EmbeddedZookeeper extends Closeable with Logging {
     // Also shuts down ZooKeeperServer
     CoreUtils.swallow(factory.shutdown(), this)
 
+    val sessionTracker: SessionTrackerImpl = zookeeper.getSessionTracker.asInstanceOf[SessionTrackerImpl]
```

Comment:
Could you please use match pattern instead of casting? That can protect casting error if the underlay instance is not `SessionTrackerImpl`